.npmignore

```diff
@@ -0,0 +1 @@
+src
```

Comment:
If I am not missing something src should not need to be included in build as it is not needed for consuming app.